domains/marco.json

```
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "marcomafessolli",
+        "email": "marcomafessolli@gmail.com"
+      },
+      "record": {
+        "CNAME": "github.com/marcomafessolli"
```

```suggestion
        "URL": "https://github.com/marcomafessolli"
```
If you want the domain to redirect to your GitHub profile, commit this suggestion.